apps/server/src/modules/learnroom/uc/dashboard.uc.ts

```diff
@@ -31,7 +56,8 @@ export class DashboardUc {
 		const dashboard = await this.dashboardRepo.getDashboardById(dashboardId);
 		this.validateUsersMatch(dashboard, userId);
 
-		dashboard.moveElement(from, to);
+		const [substituteCourses] = await this.courseRepo.findAllForSubstituteTeacher(userId);
```

Comment:
why do we need this at all when showSubstitutes is false?

Reply:
To check if this call is needed extension of endpoint with an additional parameter will be necessary. For me it doesn't make sense to extend another endpoint by showSubstitute flag because this call for courses will be removed in the nearest future as it is just temporary solution to stop arrangement and grouping of the substitute courses.